WordPress/Docs/WP/EnqueuedResourceParametersStandard.xml

```diff
@@ -0,0 +1,86 @@
+<documentation title="Enqueued Resource Parameters">
+    <standard>
+    <![CDATA[
+    The resource version must be set, to prevent the browser from retrieving an outdated, cached version after the resource has been updated.
```

Comment:
```suggestion
    The resource version must be set, to prevent the browser from using an outdated, cached version after the resource has been updated.
```

The whole point is that the browser _won't_ be retrieving the new version, but will use the locally cached version.